oligo_extract.py

```diff
@@ -28,104 +30,90 @@ def find_Oligos(config, paper_ids):
-# 'WBPaper00005783', 'WBPaper00045258', 'WBPaper00046820', 'WBPaper00002018', 'WBPaper00003969', 'WBPaper00046810',
-# 'WBPaper00024569', 'WBPaper00027336', 'WBPaper00005045', 'WBPaper00002379', 'WBPaper00047026', 'WBPaper00045803',
-# 'WBPaper00036086', 'WBPaper00041308', 'WBPaper00002121', 'WBPaper00031871', 'WBPaper00004181', 'WBPaper00024331',
-# 'WBPaper00006107', 'WBPaper00026736', 'WBPaper00028462', 'WBPaper00005275', 'WBPaper00035150', 'WBPaper00005276',
-# 'WBPaper00005955', 'WBPaper00006135', 'WBPaper00050677', 'WBPaper00005096', 'WBPaper00001768', 'WBPaper00038214',
-# 'WBPaper00036384', 'WBPaper00041364', 'WBPaper00044057',
-# 'WBPaper00032087',
-# 'WBPaper00004985',
-# 'WBPaper00031858',
-# 'WBPaper00006052',
-# 'WBPaper00026908',
-# 'WBPaper00029143',
-# 'WBPaper00002004',
-# 'WBPaper00003021',
-# 'WBPaper00048712',
-# 'WBPaper00001677',
-# 'WBPaper00005504',
-# 'WBPaper00030754',
-# 'WBPaper00004503',
-# 'WBPaper00001835',
-# 'WBPaper00030864',
-# 'WBPaper00005177',
-# 'WBPaper00004282',
-# 'WBPaper00027748',
-# 'WBPaper00003663',
-# 'WBPaper00003566',
-# 'WBPaper00001366',
-# 'WBPaper00004943',
-# 'WBPaper00027659',
-# 'WBPaper00002207',
-# 'WBPaper00001691',
-# 'WBPaper00036485',
-# 'WBPaper00049977',
-# 'WBPaper00003989',
-# 'WBPaper00032509',
-# 'WBPaper00003632',
-# 'WBPaper00003350',
-# 'WBPaper00044537',
-# 'WBPaper00048884',
-# 'WBPaper00050743'
-# 'WBPaper00050123',
-# 'WBPaper00051175',
-# 'WBPaper00051394',
-# 'WBPaper00001872',
-# 'WBPaper00003757',
-# 'WBPaper00004275',
-# 'WBPaper00004442',
-# 'WBPaper00002034',
-# 'WBPaper00005135',
-# 'WBPaper00000779',
-# 'WBPaper00005374',
-# 'WBPaper00005208',
-# 'WBPaper00005198',
-# 'WBPaper00005233',
-# 'WBPaper00045172',
-# 'WBPaper00041467',
-# 'WBPaper00006439',
-# 'WBPaper00025193',
-# 'WBPaper00001075',
-# 'WBPaper00005533',
-# 'WBPaper00040140',
-# 'WBPaper00006391']
+    # paper_ids = ["WBPaper00001366"]
+    paper_ids = ["WBPaper00003663", "WBPaper00003632", "WBPaper00003021", "WBPaper00005504", "WBPaper00030754", "WBPaper00005177", "WBPaper00004282", "WBPaper00003566", "WBPaper00001366", "WBPaper00004943", "WBPaper00002207", "WBPaper00001691", "WBPaper00003989", "WBPaper00003632", "WBPaper00044537", "WBPaper00050743", "WBPaper00001872", "WBPaper00005135", "WBPaper00000779", "WBPaper00025193", "WBPaper00005533", "WBPaper00001366", "WBPaper00002207", "WBPaper00001691",
+"WBPaper00050123",
+"WBPaper00002034",
+"WBPaper00000779",
+"WBPaper00006439",
+"WBPaper00025193",
+"WBPaper00005533",
+"WBPaper00003989",
+"WBPaper00002034",
+"WBPaper00001677",
+"WBPaper00005504",
+"WBPaper00004503",
+"WBPaper00001835",
+"WBPaper00004282",
+"WBPaper00003663",
+"WBPaper00002207",
+"WBPaper00001691",
+"WBPaper00003632",
+"WBPaper00051175",
+"WBPaper00004275",
+"WBPaper00002034",
+"WBPaper00000779",
+"WBPaper00030754",
+"WBPaper00005504",
+"WBPaper00001835"]
 
     df = find_Oligos(config, paper_ids)
-    df.to_csv('oligoss.csv', index=False)
+
```

Comment:
You can remove all the following lines if the filename for the output csv is given as a parameter. The user will take care